index.js

```diff
@@ -172,6 +188,6 @@ module.exports = (input, opts) => {
 	});
 
 	return tasks.run()
-		.then(() => readPkgUp())
+		.then(() => readPkgUp({cwd: path.join(process.cwd(), opts.contents)}))
```

Comment:
Maybe we should extract `path.join(process.cwd(), opts.contents)` to a constant at the top so we can reuse it.

Reply:
👍 Sounds good, will refactor.